### ISSUE TITLE
Could not build with Realm version 0.88.2

### ISSUE THREAD

**Issue by toanpv** [NikoYuwono/realm-browser-android#4]
> Error:Error converting bytecode to dex:
> Cause: com.android.dex.DexException: Multiple dex files define Lio/realm/RealmCache$Callback;
> :app:transformClassesWithDexForDebug FAILED

Comment this line then everything ok

> //debugCompile 'com.nikoyuwono:realm-browser:0.1'

**Comment by zetazaw** [comment]
Yes I have also error and OK after remove   //compile 'com.nikoyuwono:realm-browser:0.1'

**Comment by xiaolongyuan** [comment]
Could not build with Realm version 0.89.1
